iina/PlayerCore.swift

```diff
@@ -1089,6 +1086,11 @@ class PlayerCore: NSObject {
   func playbackRestarted() {
     Logger.log("Playback restarted", subsystem: subsystem)
     reloadSavedIINAfilters()
+
+    if info.vid == 0 {
+      notifyMainWindowVideoSizeChanged()
```

Comment:
Also, should we move this to `PlayerCore.fileLoaded`?

```diff
diff --git a/iina/PlayerCore.swift b/iina/PlayerCore.swift
index dcbb67ae..1f6b6dcb 100644
--- a/iina/PlayerCore.swift
+++ b/iina/PlayerCore.swift
@@ -1063,6 +1063,7 @@ class PlayerCore: NSObject {
 
       if self.isInMiniPlayer {
         miniPlayer.defaultAlbumArt.isHidden = !self.info.videoTracks.isEmpty
+        self.miniPlayer.updateVideoSize()
       }
     }
     // set initial properties for the first file
```
Because currently, since hiding & un-hiding the default art and `updateVideoSize` for mini player are not executed consecutively, we can see the default art shows in the original aspect ratio and then the size of the mini player got updated when we change from an audio file without art to a landscape video in music mode.